ui/components/editor/EditorMode.vue

```diff
@@ -161,9 +165,8 @@
     }
   }
 
-  & li:hover .channel-buttons,
-  & li.sortable-chosen .channel-buttons,
-  & li.sortable-ghost .channel-buttons {
+  // has to be a separate rule because older browsers would ignore the whole rule
```

Comment:
I assume Vue does support these style comments?

Reply:
This is supported by SCSS. However, as we also use the `/* ... */` comment style, we should enforce a consistent style here. It seems like there is no such rule in Stylelint yet, though.

Reply:
There are some rules regarding comments in stylelint-scss: https://github.com/kristerkari/stylelint-scss#-comment

We can either disallow `/* … */` style comments completely with [`comment-no-loud`](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/comment-no-loud/README.md) or allow `// …` style comments only as inline comments with [`double-slash-comment-inline`](https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/double-slash-comment-inline/README.md).